indieauth.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: IndieAuth
  * Plugin URI: https://github.com/indieweb/wordpress-indieauth/
  * Description: IndieAuth is a way to allow users to use their own domain to sign into other websites and services
- * Version: 3.3.2
+ * Version: 3.3.3
```

Comment:
This sounds greater than a "patch"?

Reply:
You think it should bump to 3.4? It doesn't seem to bring any new features...although I suppose the unique URL thing is a feature.

Reply:
and it breaks backwards compatibility... if only a little ;)

Reply:
Issuing corrections now.